internal/hud/renderer.go

```diff
@@ -93,9 +93,9 @@ func (r *Renderer) maybeAddAlertModal(vs view.ViewState, layout rty.Component) r
 		l.Add(rty.Fg(rty.TextString(msg), tcell.ColorDefault))
 		l.Add(rty.TextString(""))
 
-		b := rty.NewBox(l)
-		b.SetTitle("! Alert !")
-		layout = r.renderModal(rty.Fg(b, tcell.ColorRed), layout, false)
+		w := rty.NewWindow(l)
+		w.SetTitle("! Alert !")
```

Comment:
Throwaway comment: This always bugged me. Is this vaguely Spanish? Also, we are way too alarmist for something as innocuous as — say — YAML resources not having logs.

Reply:
consider it thrown away!